tools/make/packaging.mk

```diff
@@ -30,7 +30,8 @@ dist-alloy-binaries: dist/alloy-linux-amd64                    \
                      dist/alloy-darwin-amd64                   \
                      dist/alloy-darwin-arm64                   \
                      dist/alloy-windows-amd64.exe              \
-                     dist/alloy-freebsd-amd64
+                     dist/alloy-freebsd-amd64                  \
+                     dist-alloy-boringcrypto-binaries
```

Comment:
Does this need to be in this list? `make dist` already depends on `dist-alloy-boringcrypto-binaries` and I think it might be helpful to be able to separate out the two fully so you can choose to only build non-boringcrypto binaries if that's what you wanted.

Reply:
I can go either way, it feels like `dist-alloy-binaries` should create all alloy binaries but in reality only the build or us during testing is likely to want to generate that so no strong opinion.